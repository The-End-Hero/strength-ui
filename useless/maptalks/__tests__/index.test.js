import React from "react";
import assert from "power-assert";
import { render, shallow } from "enzyme";
import toJson from "enzyme-to-json";
import MapTalks from "../index";

describe("<Amap/>", () => {
  it("should render a <MapTalks/> components", () => {
    const wrapper = render(
      <div>
      </div>
    );
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
