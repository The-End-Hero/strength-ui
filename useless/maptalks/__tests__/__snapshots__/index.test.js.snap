// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Amap/> should render a <MapTalks/> components 1`] = `<div />`;
